native/react-native/common-components/modals/confirm-modal.js

```diff
@@ -0,0 +1,182 @@
+import React, { Component } from "react";
+import { Modal, Text, TouchableHighlight, View, StyleSheet } from "react-native";
+import { connect } from "react-redux";
+import { PropTypes } from "prop-types";
+import { configureConfirmModal } from "../../redux"; //This action creator is included in this folder ./modalActions.js
+
+const mapStateToProps = state => ({
+    modalState: state.modalState
+});
+
+/**
+ * Basic shape of state.modalState (referenced above in mapStateToProps), which should go in your reducer for this Redux property:
```

Comment:
We already have the initial state in code, wanna just put this comment above that and delete this duplication?

Reply:
Done!